src/Modules/Guilds/Hooks/useCurrentSnapshotId.ts

```diff
@@ -0,0 +1,25 @@
+import useGuildImplementationTypeConfig from 'Modules/Guilds/Hooks/useGuildImplementationType';

```

Comment:
Tbh, the `hooks/Guilds/ should be considered deprecated. I'd rather see the structure such as `hooks/contracts`, `hooks/ens`, `hooks/erc20`.

It will be a nasty diff though. So maybe do it in the next PR?